cf-custom-resources/lib/nlb-cert-validator-updater.js

```diff
@@ -582,10 +674,9 @@ function targetRecordExists(targetDomainName, recordSet) {
     if (!recordSet || recordSet.length === 0) {
         return false;
     }
-    return recordSet[0].Name === targetDomainName;
+    return recordSet[0].Name === `${targetDomainName}.`;
```

Comment:
🤔

Reply:
Ya... the response from route53 has the trailing dot. I could upsert the record using a domain name with or without the trailing dot - they are treated the same. It's just the response does always have the trailing dot, no matter how I upserted it in the first place.